### DISCUSSION TITLE
Epg

### DISCUSSION THREAD

**Discussion by Dariboss** [iptv-org/community#576] [Q&A] [answered]
How do I get the epg guide or to add it to my playlist

**Comment by Animenosekai** [comment, marked as answer]
You need to generate the guides yourself following the instructions in [iptv-org/epg](https://github.com/iptv-org/epg).

If you want an example, you can look at my repo [Animenosekai/japanterebi-xmltv](https://github.com/Animenosekai/japanterebi-xmltv)
